less/index.less

```diff
@@ -49,11 +49,173 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+
+
 * {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html {
+	font-size:62.5%
+	
+}
+
+body {
+	background: #fdf6e4;
+	
+}
+
+.container {
+	font-size:1.6rem;
 }
 
+.container {
+	width: 880px;
+	max-width: 800px;
+	min-width: 660px;
+	margin: 0 auto;
+	border: 1px solid black;
+	.main-nav {
+		background: #2071c8;
+		border: 1px solid black;
+		height: 13vh;
```

Comment:
careful with vh, this will cause the nav to basically disappear as screens get flattened out